Minimal definition of what all GEDCOM files share (#50)
FamilySearch/GEDCOM: version-detection/version-detection.md

a parser can detect the correct version and parse the file accordingly.  This document specifies a version
detection algorithm and provides references to the more specific format specifications.

## Commonalities

> All version of GEDCOM files are text files that conform to the following general structure:

Reviewer: ```suggestion
All versions of GEDCOM files are text files that conform to the following general structure:
```

> - The document is organized as a sequence of **lines** separated by ASCII line terminators (U+000A, U+000D, or both). Blank lines are ignored.

Reviewer: ```suggestion
- The file is organized as a sequence of **lines** separated by line terminators (U+000A, U+000D, or both).
```

> - Each line contains several components, separated by spaces (U+0020):

Reviewer: ```suggestion
- Each non-blank line contains several components, separated by spaces (U+0020):
```

  1. A required **level**, which is a non-negative integer encoded using ASCII digits.
    - The first line's level is 0.
    - Each other line's level is at most 1 greater than the level of the line before it.
  2. An optional **identifier**, which begins and ends with a COMMERCIAL AT (U+0040 `@`) and contains no internal line terminators or COMMERCIAL ATs. No two lines share the same identifier.
  3. A required **tag**, which is a string of one or more ASCII digits (U+0030 through U+0039 `9`–`0`), letters (U+0041 through U+005A `A`–`Z` and U+0061 through U+007A `a`–`z`), and underscores (U+005F `_`).
  4. An optional **value**, which may contain any non-line terminator character.

> - Each line represents a structure. Any line with level *x* > 0 represents a substructure of the structure represented by the nearest preceding line with level *x* − 1.

Reviewer: The term "structure" here is not defined, and does not seem essential to understanding this sentence.   Here's an attempt to phrase in more layman's terms:
```suggestion
- Any line with level *x* > 0 represents a hierarchical child of the nearest preceding line with level *x* − 1.
```
